pkg/flavor/kubernetes/flavor.go

```diff
@@ -45,6 +45,8 @@ const (
 	chapSecretNamespaceKey        = "chapSecretNamespace"
 	chapUserKey                   = "chapUser"
 	chapPasswordKey               = "chapPassword"
+	chapSecretNameEnvKey          = "CHAP_SECRET_NAME"
+	chapSecretNamespaceEnvKey     = "CHAP_SECRET_NAMESPACE"
 	chapUserValidationPattern     = "^[a-zA-Z0-9][a-zA-Z0-9\\-:.]{0,63}$"
 	chapPasswordValidationPattern = "^$|^[a-zA-Z0-9+_)(*^%$#@!]{12,16}$"
```

Comment:
This regex doesn't contain the `-` either (as in co-deployments)

Reply:
Updated